src/System.Text.Json/src/System/Text/Json/JsonEncodedText.cs

```diff
@@ -35,38 +36,40 @@ private JsonEncodedText(byte[] utf8Value)
         /// Encodes the string text value as a JSON string.
         /// </summary>
         /// <param name="value">The value to be transformed as JSON encoded text.</param>
+        /// <param name="encoder"></param>
```

Comment:
I only fixed the build break by adding these but didn't add the actual description, make sure you do that before merging.

Reply:
Yep, will do. Thanks.